### PR TITLE
Create reset command for user management

### PR DESCRIPTION
**Pull request by ivov**
```
./packages/cli/bin/n8n user-management:reset
```
Pinned chokidar to fix build: https://github.com/n8n-io/n8n/pull/2696

### CODE REVIEW

File: packages/cli/commands/user-management/reset.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		LoggerProxy.init(logger);
+		await Db.init();
+
+		try {
+			const globalRole = await Db.collections.Role!.findOne({
+				name: 'owner',
+				scope: 'global',
+			});
+
+			const instanceOwner = await Db.collections.User!.findOneOrFail({ globalRole });
```
Comment (krynble) [inline packages/cli/commands/user-management/reset.ts:25]: I think we should add a try/catch block here and create an owner if it does not exist.

This is to make this command 100% bulletproof i.e. if someone truncates the user table (for whatever reason), we create a new owner and they can once again use the instance.

File: packages/cli/commands/user-management/reset.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				{ user: instanceOwner },
+			);
+
+			// delete all users from users table except owner
+			await Db.collections.User!.delete({ id: Not(instanceOwner.id) });
+
+			// reset user to being a shell
+			await Db.collections.User!.save(
+				Object.assign(instanceOwner, {
+					firstName: null,
```
Comment (krynble) [inline packages/cli/commands/user-management/reset.ts:45]: The initial migration sets first and last names to `default`. I think we should remain consistent, changing either here or the initial migration.

File: packages/nodes-base/package.json [hunk 3]
```diff
@@ -698,7 +698,7 @@
     "basic-auth": "^2.0.1",
     "change-case": "^4.1.1",
     "cheerio": "1.0.0-rc.6",
-    "chokidar": "^3.5.2",
+    "chokidar": "3.5.2",
```
Comment (krynble) [inline packages/nodes-base/package.json:701]: What is this change? Could you add a description why it's needed to the PR?

**Review by krynble — CHANGES_REQUESTED** [review]
Hey Ivan!

Loved that you used TypeORM operations smartly here, I learned a few new tricks.

Added some notes.

Thanks!

File: packages/nodes-base/package.json (hunk 3 above)
Comment (ivov) [reply packages/nodes-base/package.json:701]: To fix the build. Fixed it yesterday in `master`: https://github.com/n8n-io/n8n/pull/2696

**Review by krynble — APPROVED** [review]

**Pull request merged**